kmip/tests/integration/services/test_integration.py

```diff
@@ -419,3 +487,75 @@ def test_symmetric_key_register_get_destroy(self):
         observed = result.result_reason.enum
 
         self.assertEqual(expected, observed)
+
+    def test_symmetric_key_register_wrapped_get_destroy(self):
+        key_name = 'Integration Test - DEK'
+        result = self._create_symmetric_key(key_name=key_name)
+        self._check_result_status(result, ResultStatus, ResultStatus.SUCCESS)
+        self._check_uuid(result.uuid.value, str)
+        dek_uuid = result.uuid.value
+
+        key_name = 'Integration Test - KEK'
+        result = self._create_symmetric_wrapping_key(key_name=key_name)
+        self._check_result_status(result, ResultStatus, ResultStatus.SUCCESS)
+        self._check_uuid(result.uuid.value, str)
+        kek_uuid = result.uuid.value
+
+        kws = {  # key wrapping specification
+            'wrapping_method': WrappingMethod(WrappingMethodEnum.ENCRYPT),
+            'encryption_key_information': EncryptionKeyInformation(
+                unique_identifier=UniqueIdentifier(kek_uuid)),
+            "attribute_name": KeyWrappingSpecification.AttributeName(
+                value="Cryptographic Usage Mask")}
+
+        # get and verify the unwrapped DEK
+        dek_clear_result = self.client.get(uuid=dek_uuid, credential=None,
+                                           key_wrapping_specification=None)
+        self._check_result_status(dek_clear_result, ResultStatus,
+                                  ResultStatus.SUCCESS)
+
+        # get and verify the wrapped DEK
+        dek_wrapped_result = self.client.get(uuid=dek_uuid, credential=None,
+                                             key_wrapping_specification=kws)
+        self._check_result_status(dek_wrapped_result, ResultStatus,
+                                  ResultStatus.SUCCESS)
+
+        self.assertEqual(dek_clear_result.uuid.value,
+                         dek_wrapped_result.uuid.value)
+
+        clear_key = (dek_clear_result.secret.key_block.key_value.
+                     key_material.value)
+
+        wrapped_key = (dek_wrapped_result.secret.key_block.key_value.value)
+
+        # the wrapped key data should not be the same as the clear key data
+        self.assertNotEqual(clear_key, wrapped_key)
```

Comment:
This proves that the two keys are not equal, but I would like to see proof that the key was encrypted correctly. Can you add code to unencrypt the wrapped key and then compare to the clear key? This would prove to me that the key was successfully encrypted.